test/spec/modules/medianetBidAdapter_spec.js

```diff
@@ -590,14 +589,14 @@ describe('Media.net bid adapter', () => {
       documentStub.withArgs('div-gpt-ad-1460505748561-0').returns({
         getBoundingClientRect: () => boundingRect
       });
+
       let bidReq = spec.buildRequests(VALID_BID_REQUEST, VALID_AUCTIONDATA);
       let data = JSON.parse(bidReq.data);
       expect(data.imp[0].ext.visibility).to.equal(2);
       expect(data.imp[0].ext.viewability).to.equal(0);
-      sandbox.restore();
+      documentStub.restore();
```

Comment:
Since the `documentStub` was created with the sandbox it should be cleaned up by the sandbox automatically by the `sandbox.restore()` in the `after`.  Doing `documentStub.restore()` might double restore, which can do weird things.